volume-modifier-for-k8s.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: volume-modifier-for-k8s
-  version: 0.2.1
-  epoch: 8
+  version: 0.3.1
```

Comment:
it looks like they didn't release the 0.3.x version stream yet, so I think we shouldn't be doing that here meaning bumping to non-released version of a package yet:

- https://github.com/awslabs/volume-modifier-for-k8s/releases

Reply:
Good point. At the same time the official Helm chart for the [EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) already pins the v0.3.0 since the version v1.30 of the chart:
- https://github.com/kubernetes-sigs/aws-ebs-csi-driver/commit/78d76d53d9bc72827112a4e1d27496250d6e0c8b#diff-cd438cefb202fb6fb653f659771ea635aa04dd99bb6d0902a0ab8eabebcf0a1eR147

Do we want to not be able to support it?

Reply:
JFI I explicitely asked to the upstream project: https://github.com/awslabs/volume-modifier-for-k8s/issues/34

Reply:
Project uses a rolling release with tags here: https://github.com/awslabs/volume-modifier-for-k8s/tags